### PR TITLE
Relax dependency constraints

### PR DESCRIPTION
**Pull request by ofek**
@i80and Can you please take a look? We want to use this but cannot until a version is released with looser dependency constraints.

### CODE REVIEW

**Comment by i80and** [comment]
@ofek Hello! I'm happy to relax most of these constraints, but we will have to continue to pin `docutils==0.16` since we interact closely with its internals.

File: pyproject.toml [hunk 1]
```diff
@@ -11,11 +11,11 @@ classifiers = ["License :: OSI Approved :: Apache Software License"]
 requires-python = "~= 3.7"
 requires = [
     "pymongo[srv,tls] ~= 3.11.1",
-    "docutils == 0.16",
-    "watchdog ~= 1.0.1",
-    "requests ~= 2.24.0",
-    "toml ~= 0.10.1",
-    "pyyaml ~= 5.3.1",
+    "docutils ~= 0.16",
```
Comment (ofek) [inline pyproject.toml:14]:
```suggestion
    "docutils == 0.16",
```

**Comment by ofek** [comment]
Done!

**Review by i80and — APPROVED** [review]

**Comment by ofek** [comment]
@i80and Thanks, I'll keep a look out for the next release!

**Pull request merged**